lib/puppet/defaults.rb

```diff
@@ -1926,11 +1926,11 @@ def self.default_vendormoduledir
     },
     :ldapserver => {
       :default  => "ldap",
-      :desc     => "The LDAP server.  Only used if `node_terminus` is set to `ldap`.",
+      :desc     => "The LDAP server.",
     },
     :ldapport => {
       :default  => 389,
-      :desc     => "The LDAP port.  Only used if `node_terminus` is set to `ldap`.",
```

Comment:
Why are we only removing part of the description here? Are these defaults still being used after removing the terminus?

Reply:
Yes actually.  There is a remaining LDAP provider (lib/puppet/provider/ldap.rb) which uses the LDAP manager (lib/puppet/util/ldap/manager.rb), which then uses the LDAP connection (lib/puppet/util/ldap/connection.rb).  Both the manager and the connection use the ldapserver and ldapport settings.